include/aspect/material_model/interface.h

```diff
@@ -765,7 +765,7 @@ namespace aspect
         /**
          * Destructor.
          */
-        virtual ~NamedAdditionalMaterialOutputs();
+        ~NamedAdditionalMaterialOutputs() override;
```

Comment:
Isn't our usual style to leave the `virtual` in place even when we add `override` to the function?

Reply:
I don't know about "our usual style" with regard to that. I enabled ``modernize-use-override`` in #3249, which flags these places. Can you take a look at #3258?

Reply:
The C++ core guidelines specify that there should be exactly one keyword: https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#c128-virtual-functions-should-specify-exactly-one-of-virtual-override-or-final